payroll/services.py

```diff
@@ -128,7 +128,11 @@ def _get_bills_queryset(self, obj_data, included_unpaid):
             date_bill__range=(date_from, date_to),
             subject_type=ContentType.objects.get_for_model(Beneficiary),
             subject_id__in=beneficiary_ids,
-            status__in=[Bill.Status.VALIDATED]
+            status__in=[Bill.Status.VALIDATED],
+        )
+
+        bills_queryset = bills_queryset.filter(
+            Q(payrollbill__isnull=True) | Q(payrollbill__is_deleted=True)
```

Comment:
Why payroll__is_deleted=True? 
I think that when we delete payroll, all bills are detached and moreover PayrollBill entity is completely removed. Can you confirm @jdolkowski